src/firestore/index.ts

```diff
@@ -75,58 +75,72 @@ export {
 export { FirestoreSettings };
 
 /**
- * Gets the {@link https://googleapis.dev/nodejs/firestore/latest/Firestore.html | Firestore}
+ * Gets the default {@link https://googleapis.dev/nodejs/firestore/latest/Firestore.html | Firestore}
  * service for the default app.
  *
- * `getFirestore()` can be called with no arguments to access the default
```

Comment:
Please get these documentation changes reviewed by a tech writer before merging the PR. Thanks!

Reply:
@markarndt

Reply:
Thanks Mark!